sda-pipeline/cmd/intercept/intercept.go

```diff
@@ -124,7 +124,7 @@ func main() {
 			}
 
 			routing := map[string]string{
-				msgAccession: "accessionIDs",
+				msgAccession: "accession",
```

Comment:
Is that true that the name `accessionIDs` used only for the queue name?

Reply:
`Mixed case queue names is not allowed anymore`